src/helpers/deserializer.ts

```diff
@@ -6,6 +6,8 @@ import { JSONObject } from './json';
 
 export type DeserializableResponse = AxiosResponse<Deserializer>;
 
+export type DeserializableArrayResponse = AxiosResponse<Deserializer[]>;
```

Comment:
Should we use the `List` to make it consistent?
```suggestion
export type DeserializableListResponse = AxiosResponse<Deserializer[]>;
```